test/integration/ads_integration_test.cc

```diff
@@ -236,6 +236,93 @@ TEST_P(AdsIntegrationTest, ClusterInitializationUpdateOneOfThe2Warming) {
   test_server_->waitForGaugeGe("cluster_manager.active_clusters", 4);
 }
 
+TEST_P(AdsIntegrationTest, DeltaSdsRemovals) {
+  // This test is for delta only
```

Comment:
Please add a comment why this is only relevant to Delta-xDS and not SotW.

Also comments style-guide, comments need to end with '.'.

Reply:
Ack; done!